apps/hash-frontend/src/pages/admin/admin-page-layout.tsx

```diff
@@ -39,6 +45,21 @@ const AdminLayout = ({ children }: PropsWithChildren) => {
     [router],
   );
 
+  const { isInstanceAdmin } = useAuthenticatedUser();
+
+  useEffect(() => {
+    /**
+     * Redirect non instance admins away from `/admin` pages
+     */
+    if (typeof isInstanceAdmin === "boolean" && !isInstanceAdmin) {
```

Comment:
Nit, can merge it as it is
```suggestion
    if (isInstanceAdmin === false) {
```